lnwallet/channel.go

```diff
@@ -2406,6 +2406,16 @@ func (lc *LightningChannel) evaluateHTLCView(view *htlcView, ourBalance,
 		}
 
 		addEntry := lc.remoteUpdateLog.lookupHtlc(entry.ParentIndex)
+		if addEntry == nil {
```

Comment:
No issue with this addition as is, but we'll want to remove this once we find the source of the bug itself.